cfgov/jinja2/v1/_includes/organisms/form-explainer.html

```diff
@@ -0,0 +1,198 @@
+
+   ==========================================================================
+
+   Description:
+
+   Render a list of o-expandable terms with definitions.
+
+   notes:    An array of note objects.
+
+   page_num: The page that these notes belong to.
+
+   category_index: category that these notes belong to.
+
+   ========================================================================== #}
+
+{% macro render_terms( notes, page_num, category_index ) %}
+{% for item in notes %}
+    {{ render_term( item, page_num, category_index, loop.index ) }}
+{% endfor %}
+{% endmacro %}
+
+
+{# ==========================================================================
+
+   render_term( note, page_num, category_index, note_index )
+
+   ==========================================================================
+
+   Description:
+
+   Render an o-expandable note with a definition.
+
+   item:      A note object. Note objects have the following structure:
+              {
+                'heading': 'Note title.',
+                'body':    'Text of note.'
+              }
+
+   page_num:   Page containing note.
+
+   category_index: Category containing note.
+
+   note_index: index of note in category note array.
+
+
+   ========================================================================== #}
+
+{% macro render_term( item, page_num, category_index, note_index ) %}
+{% set id = 'page-' + page_num | string + '-category-' + category_index | string + '-note-'  + note_index | string %}
+<div class="o-expandable
+            o-expandable__padded
+            o-expandable__form-explainer
+            o-expandable__form-explainer-category-{{ category_index }}"
+     id="{{ id }}">
+    <button class="o-expandable_header o-expandable_target"
+            tabindex="0">
+        <span class="h4 o-expandable_header-left o-expandable_label">
+            {{ item.heading }}
+        </span>
+        <span class="o-expandable_header-right o-expandable_link">
+            <span class="o-expandable_cue-open" aria-label="Show">
+                {{ svg_icon('plus-round') }}
+            </span>
+            <span class="o-expandable_cue-close" aria-label="Hide">
+                {{ svg_icon('minus-round') }}
+            </span>
+        </span>
+    </button>
+    <div class="o-expandable_content" tabindex="0">
+        {{ item.body | safe }}
+    </div>
+</div>
+{% endmacro %}
+
+
+{# ==========================================================================
+
+   render_pagination( pages )
+
+   ==========================================================================
+
+   Description:
+
+   Render pagination.
+
+   pages: Pages in form. The data structure is as follows:
+          [
+            { 'image':   wagtail image,
+              'categories': [ ... ]
+            }
+          ]
+
+   ========================================================================== #}
+
+{% macro render_pagination( pages ) %}
+<nav class="pagination explain_pagination">
```

Comment:
Maybe for a future PR but this should be updated to use `m-pagination` https://cfpb.github.io/design-system/components/pagination

Reply:
@anselmbradford Yes, that's in the works for a future PR!